main_routers/cloudsave_router.py

```
@@ -0,0 +1,421 @@
+    is_cloudsave_provider_available,
+    restore_cloudsave_operation_backup,
+)
+
+
+router = APIRouter(prefix="/api/cloudsave", tags=["cloudsave"])
+
+
+def _decode_json_response_payload(response: JSONResponse) -> dict:
+    try:
+        return json.loads(response.body.decode("utf-8"))
+    except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)